app/screens/receive-bitcoin-screen/receive-usd.tsx

```diff
@@ -443,13 +442,17 @@ const TimeInformation = ({
     return <></>
   }
 
+  const date = new Date(timeLeft * 1000)
+  const minutes = date.getUTCMinutes()
+  const seconds = date.getUTCSeconds()
+
   return (
     <View style={styles.countdownTimer}>
       <Text style={timeLeft < 10 ? styles.lowTimer : undefined}>
         {LL.ReceiveWrapperScreen.expiresIn()}:{" "}
       </Text>
       <View style={styles.timer}>
-        <Text>{moment.utc(timeLeft * 1000).format("m:ss")}</Text>
+        <Text>{`${minutes}:${seconds}`}</Text>
```

Comment:
Will this result in weird formatting for seconds that are less than 10? For example 2:08 would be formatted as 2:8?

Reply:
mmm probably. good catch. will try and fix if this is the case